src/session/sessionStore.ts

```diff
@@ -46,27 +47,36 @@ export const inferSessionQuery = (
   if (!req.url) {
     return
   }
-  const url = new URL(req.url)
```

Comment:
Actually here for me it always crashed because `req.url` was a relative path (e.g.:`/?queryParam=1`) that is why I introduced a new function `getQueryFromUrl` to just retrieve the queryString of the URL and then I pass that to `URLSearchParams`

Reply:
Let me know what you think @eunjae-lee

Reply:
Sorry you're right. Let's bring it back